04.ls/ls.rb

```diff
@@ -55,71 +65,75 @@ def check_stickybit_sgid_suid(oct_mode, permission)
 end
 
 def get_permission(oct_mode)
-  permission_data = { '0' => '---', '1' => '--x', '2' => '-w-', '3' => '-wx', '4' => 'r--', '5' => 'r-x', '6' => 'rw-', '7' => 'rwx' }
+  permission_data = { '0' => '---',
+                      '1' => '--x',
+                      '2' => '-w-',
+                      '3' => '-wx',
+                      '4' => 'r--',
+                      '5' => 'r-x',
+                      '6' => 'rw-',
+                      '7' => 'rwx' }
```

Comment:
括弧と中身は別の行にした方が良いです。変数名や最初の要素、最後の要素を修正する時に要らぬバグをうむ可能性があるので。
```suggestion
  permission_data = {
                      '0' => '---',
                      '1' => '--x',
                      '2' => '-w-',
                      '3' => '-wx',
                      '4' => 'r--',
                      '5' => 'r-x',
                      '6' => 'rw-',
                      '7' => 'rwx'
}
```

Reply:
この観点は全くなかったです。教えていただいてありがとうございます🙌